public/app/features/alerting/unified/components/rule-editor/NotificationsStep.tsx

```diff
@@ -56,7 +62,14 @@ export const NotificationsStep = ({ alertUid }: NotificationsStepProps) => {
       }
       fullWidth
     >
-      <LabelsField dataSourceName={dataSourceName} />
+      <LabelsFieldInForm onEditClick={onEditClick} />
+      {showLabelsEditor && (
```

Comment:
Do we need the `showLabelsEditor` condition?
I think if we pass `false` to the `isOpen` prop, the modal won't render

Reply:
you're right 👍